test/templates/cluster_endtoend_test.tpl

```diff
@@ -0,0 +1,49 @@
+name: {{.Name}}
+on: [push, pull_request]
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
```

Comment:
@rohit-nayak-ps 
Can we add to the template something like
```
name: Run endtoend tests on {{.Name}}
```
It turns out that once the PR is merged the top-level name is not displayed any more, only the name from here is displayed and it defaults to `build`.

Reply:
Created #7289